code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/SubmissionsGraph.kt

```diff
@@ -0,0 +1,29 @@
+package org.jetbrains.research.code.submissions.clustering
+
+import com.jetbrains.rd.util.string.printToString
+import org.jetbrains.research.code.submissions.clustering.load.Unifier
+import org.jgrapht.Graph
+import org.jgrapht.graph.DefaultWeightedEdge
+import org.jgrapht.graph.SimpleDirectedWeightedGraph
+
+class SubmissionsGraph(private val graph: Graph<Submission, DefaultWeightedEdge>) {
+    fun print() = graph.printToString()
```

Comment:
`printToString` function really prints the graph into the console or just returns its seeing representation? maybe it is better to give `buildStringRepresentation` instead of `print`?